src/govuk/components/footer/footer.yaml

```diff
@@ -117,6 +143,14 @@ examples:
   data:
     {}
 
+- name: with custom content licence and copyright notice
```

Comment:
Would be good to have text and html examples of each, so we're (hopefully) more like to catch [issues like this](https://github.com/alphagov/govuk-frontend/issues/2703).

Reply:
What would be the best way to present this? Have two examples in the review app with the same text content but one passed through `html` and the other with `text`?

Reply:
Done 👍